src/Features/Traits/HttpResponse.php

```diff
@@ -0,0 +1,251 @@
+        foreach ($table->getRowsHash() as $key => $value) {
+            Assert::assertArrayHasKey($key, $json);
+            Assert::assertStringContainsString($value, $json[$key]);
+        }
+    }
+
+    /**
+     * @Then the response should have cookie :name
+     */
+    public function aResponseShouldHaveCookie(string $name): void
+    {
+        Assert::assertTrue($this->response->headers->has("Set-Cookie"));
+        $cookies = $this->response->headers->getCookies();
+        $cookieNames = array_map(fn($cookie) => $cookie->getName(), $cookies);
+        Assert::assertContains($name, $cookieNames);
+    }
+
+    /**
+     * @Then the response should have header :header
+     */
+    public function aResponseShouldHaveHeader(string $header): void
+    {
+        Assert::assertTrue($this->response->headers->has($header));
+    }
+
+    /**
+     * @Then the response should have header :header equal to :value
+     */
+    public function aResponseShouldHaveHeaderEqualTo(string $header, string $value): void
+    {
+        Assert::assertEquals($value, $this->response->headers->get($header));
+    }
+
+    /**
+     * @Then the response should have status :status
+     */
+    public function aResponseShouldHaveStatus(int $status): void
+    {
+        Assert::assertEquals($status, $this->response->getStatusCode());
+    }
+
+    /**
+     * @Then the response should be OK
+     */
+    public function aResponseShouldBeOk(): void
+    {
+        Assert::assertEquals(Response::HTTP_OK, $this->response->getStatusCode());
+    }
+
+    /**
+     * @Then the response should be created
+     */
+    public function aResponseShouldBeCreated(): void
+    {
+        Assert::assertEquals(Response::HTTP_CREATED, $this->response->getStatusCode());
+    }
+
+    /**
+     * @Then the response should be no content
+     */
+    public function aResponseShouldBeNoContent(): void
+    {
+        Assert::assertEquals(Response::HTTP_NO_CONTENT, $this->response->getStatusCode());
+    }
+
+    /**
+     * @Then the response should be bad request
+     */
+    public function aResponseShouldBeBadRequest(): void
+    {
+        Assert::assertEquals(Response::HTTP_BAD_REQUEST, $this->response->getStatusCode());
+    }
+
+    /**
+     * @Then the response should be unauthorized
+     */
+    public function aResponseShouldBeUnauthorized(): void
+    {
+        Assert::assertEquals(Response::HTTP_UNAUTHORIZED, $this->response->getStatusCode());
+    }
+
+    /**
+     * @Then the response should be forbidden
+     */
+    public function aResponseShouldBeForbidden(): void
+    {
+        Assert::assertEquals(Response::HTTP_FORBIDDEN, $this->response->getStatusCode());
+    }
+
+    /**
+     * @Then the response should be not found
+     */
+    public function aResponseShouldBeNotFound(): void
+    {
+        Assert::assertEquals(Response::HTTP_NOT_FOUND, $this->response->getStatusCode());
+    }
+
+    /**
+     * @Then the response should be conflict
+     */
```

Comment:
@EwelinaSkrzypacz, what do you think? Maybe `the response status should be conflict` would be better?

Reply:
I agree with that.

Reply:
Okay, changed